test/groovy/MulticloudDeployTest.groovy

```diff
@@ -95,6 +87,9 @@ class MulticloudDeployTest extends BasePiperTest {
                 ]
             ]
         ]
+
+        helper.registerAllowedMethod('echo', [CharSequence.class], {s -> println(s)})
```

Comment:
Debug output?

Reply:
No, what would you suggest? I think it is ok to print echos in a test.

Reply:
In my personal opinion there is no benefit when printing info during tests at all. Nobody will read that normally.  In case some log output (remember perform logging in groovy basically with `echo`) is important it should be asserted. I vote for either removing the `println`. When everybody does that we end up in a huge amount of test logs which are not helpful at all.